### ISSUE TITLE
[BUG] Aspect ratio of image

### ISSUE THREAD

**Issue by Artmadiar** [Darginec05/Yoopta-Editor#136]
### Has this bug been raised before?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Description

Image has 3 states of how to render image: Fit, Fill and Cover.
None of this doesn't keep the original size of the image.
It's a nice idea but it requires more investigation and variants.
I hope we'll discuss it here

![3 variants of object-fit](https://github.com/Darginec05/Yoopta-Editor/assets/3044387/b74f3dba-a0b1-4dbf-82ae-fd5b17f05526)

_Those "white" areas around image is really annoying_
Example of "not standard" image (I mean the aspect ratio that differs from document width)
![image 2000x1000](https://github.com/Darginec05/Yoopta-Editor/assets/3044387/0e4ecbb2-259e-478e-ae4f-8cee6444d315)

### Steps to Reproduce

If you know the steps, follow the below format and provide steps to reproduce

For example:

1. Add new image block
2. Upload new image with not trivial aspect ratio, for example let's take resolution 2000x1000
3. Apply any aspect ratio

### Environment

- OS: MacOS 13.6.6 (22G630)
- Browser: Version 124.0.6367.119 (Official Build) (arm64)

### Do you want to work on this issue?

Yes

### If "yes" to above, please explain how you would technically implement this

I would split this task in two steps:
1. Just keep original image aspect ratio
2. Discuss the styles for each variant and add it in options

**Comment by Artmadiar** [comment]
I guess the root of the problem is that image wrapper has own styles and the image inside it has own styles:
and they have different rules of aspect ratio

**Comment by Darginec05** [comment]
@Artmadiar That's because Image plugin have `maxSizes` option. 
By default it has `maxSizes: { maxWidth: 650, maxHeight: 550 }`, but you can rewrite this behaviour: 
```
Image.extend({
  options: {
    maxSizes: {
      maxHeight:<value>,
      maxWidth: <value>,
    },
  },
});
```

**Comment by Artmadiar** [comment]
@Darginec05 Is it possible no set maxHeight to null to ignore limits in height?
I need only maxWidth

**Comment by Darginec05** [comment]
I have added the ability to return "fit" option for the required "onUpload" function in Image and Video plugins.
For your needs, you can return "fit: cover" in the "onUpload" function.
Example below: 

```
  Image.extend({
    options: {
      onUpload: async (file: File) => {
        const data = await uploadToCloudinary(file);

        return {
          src: data.secure_url,
          alt: 'cloudinary',
         // new added field
          fit: 'cover',
          sizes: {
            width: data.width,
            height: data.height,
          },
        };
      },
    },
  }),
```

**Comment by Darginec05** [comment]
Available in v4.2.0
